client/app/components/datagrid/styles.ts

```diff
@@ -0,0 +1,92 @@
+import { BC_GOV_BACKGROUND_COLOR_BLUE } from "@/app/styles/colors";
+
+// DataGrid component styles
+const styles = {
+  "& .MuiDataGrid-columnHeaderDraggableContainer": {
+    minWidth: "100%",
+  },
+  "& .MuiDataGrid-row": {
+    // Couldn't find a way to set minHeight without using !important
+    // This was needed to use getRowHeight={() => "auto"} prop to break long text into multiple lines
+    minHeight: "60px!important",
+  },
+  "& .MuiDataGrid-columnHeader": {
+    border: "1px white solid",
+    borderBottom: "none",
+    borderTop: "none",
+    color: "white",
+    backgroundColor: BC_GOV_BACKGROUND_COLOR_BLUE,
+    fontWeight: "bold",
+  },
+  "& .MuiDataGrid-columnHeadersInner": {
```

Comment:
This `sx` prop in `DataGrid.tsx` was getting rather large so I've moved it to it's own file and spread it in the `sx` style prop. Dynamic styles go underneath the spread.